source/arroost/entities/arrows/recording.js

```diff
@@ -67,19 +67,19 @@ export class ArrowOfRecording extends Entity {
 		// Style elements
 
 		this.use(() => {
-			const hasSound = this.hasSound.get()
-			if (hasSound) {
-				// Gets bigger when at lower pitch
-				const scale = -this.pitch.get() / 1000
-				const clampedScale = Math.max(-10 / 11 / 2, Math.min(10 / 11 / 2, scale))
-				this.back.dom.transform.scale.set([1, 1])
-				this.front.dom.transform.scale.set([1 / 2 + clampedScale, 1 / 2 + clampedScale])
-			} else {
-				this.back.dom.transform.scale.set([2 / 3, 2 / 3])
-				this.front.dom.transform.scale.set([1 / 3, 1 / 3])
-			}
+			// Gets bigger when at lower pitch
+			const scale = -this.pitch.get() / 1000
+			const clampedScale = Math.max(-10 / 11 / 2, Math.min(10 / 11 / 2, scale))
+			this.back.dom.transform.scale.set([1, 1])
+			this.front.dom.transform.scale.set([1 / 2 + clampedScale, 1 / 2 + clampedScale])
```

Comment:
I'm making slots look 'small'. So I can't use that as a placeholder for non-sound recordings anymore.